commons/src/main/java/com/powsybl/commons/report/BundleMessageTemplateProvider.java

```diff
@@ -26,7 +26,7 @@ public BundleMessageTemplateProvider(String bundleBaseName) {
 
     public BundleMessageTemplateProvider(String bundleBaseName, boolean throwIfUnknownKey) {
         this.bundleBaseName = bundleBaseName;
-        this.resourceBundle = ResourceBundle.getBundle(bundleBaseName, ReportConstants.DEFAULT_LOCALE);
+        this.resourceBundle = ResourceBundle.getBundle(bundleBaseName, ReportConstants.getDefaultLocale());
```

Comment:
```suggestion
        this.resourceBundle = ResourceBundle.getBundle(bundleBaseName, ReportConstants.getDefaultLocale(), NO_FALLBACK_CONTROL);
```
With 
```java
    public static final ResourceBundle.Control NO_FALLBACK_CONTROL = ResourceBundle.Control.getNoFallbackControl(ResourceBundle.Control.FORMAT_DEFAULT);
```

And similarly in getTemplate a few lines after

Reply:
done